README.md

```diff
@@ -0,0 +1,72 @@
+[![CI Build Status](https://github.com/kiwix/kiwix-apple-custom/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/kiwix/apple/actions/workflows/ci.yml?query=branch%3Amain)
+[![CD Build Status](https://github.com/kiwix/kiwix-apple-custom/actions/workflows/cd.yml/badge.svg?branch=main)](https://github.com/kiwix/kiwix-apple-custom/actions/workflows/cd?query=branch%3Amain)
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
+
+## Custom app folder
+
+[In the repository](https://github.com/kiwix/kiwix-apple-custom),
+each custom configuration is isolated in a so called custom app
+folder. If you need to create a new one for a new custom app, please use a folder name
+that is lowercased and contains no space.
+
+## info.json file
+
+The configuration of the custom app is handled using the `info.json`
+file which **is required to be placed in the custom app folder**. Take example on an already
+existing one if you need to create a new custom app. The most
+important fields are:
+- `about_app_url` - this is an external link that is placed in the "About section" of the application. (Eg. "https://www.dwds.de")
+- `about_text` - this is a custom text that is placed in the "About section" describing what the application is about.
+- `app_name` - the title of the app
+- `app_store_id` - this should to be taken from the developer.apple.com, where the application release is prepared. Note you can take use the app_store_id even if the app is not yet released. The id is used within the app in the "Rate the app" section, so users can be redirected to a specific app in the App Store, to rate it.
+- `build_version` - it should be a single digit number, used as the last part of the final app version.
+[MAJOR].[MINOR].**[PATCH]**. The final version of the app contains the date of the ZIM file provided (using the year and month) and the build_version as the last part. The format of the version comes from the [Apple docs](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleshortversionstring). Eg., given a ZIM file: "dictionary_nopic_2023-12-15.zim", and "build_version": "5" we combine it the following way: take the year 2023 and the month 12 from the ZIM file, and combine it with the build_version, resulting in: "2023.12.5". [More on this process here](https://github.com/kiwix/apple/issues/559).
+- `enforced_lang` - (lang code eg: en, de, he) if it is set, it will include only this language in the final app, meaning no other languages can be selected (on iOS) for the application UI. See the current list of supported languages [already translated in the main repo](https://github.com/kiwix/apple/tree/main/Support). When using this option, make sure that [the translation coverage](https://translatewiki.net/wiki/Special:MessageGroupStats/kiwix-apple?group=kiwix-apple&messages=&suppressempty=1&x=D) is 100% for the enforced language.
```

Comment:
```suggestion
- `enforced_lang` - ISO 639-1 language code (eg: en, de, he) if it is set, it will include only this language in the final app, meaning no other languages can be selected (on iOS) for the application UI. See the current list of supported languages [already translated in the main repo](https://github.com/kiwix/apple/tree/main/Support). When using this option, make sure that [the translation coverage](https://translatewiki.net/wiki/Special:MessageGroupStats/kiwix-apple?group=kiwix-apple&messages=&suppressempty=1&x=D) is 100% for the enforced language.
```